crates/sui-types/src/quorum_driver_types.rs

```diff
@@ -8,15 +8,31 @@ use crate::base_types::{AuthorityName, ObjectRef, TransactionDigest};
 use crate::committee::StakeUnit;
 use crate::crypto::ConciseAuthorityPublicKeyBytes;
 use crate::error::SuiError;
-use crate::messages::{QuorumDriverResponse, VerifiedTransaction};
+use crate::messages::{
+    QuorumDriverResponse, TransactionEvents, VerifiedCertifiedTransactionEffects,
+    VerifiedTransaction,
+};
+use crate::object::Object;
 use serde::{Deserialize, Serialize};
 use strum::AsRefStr;
 use thiserror::Error;
 
+// QuorumDrvierResponse with verified input Objects passed back from
+// Validator responses for fast execution.
+#[derive(Debug, Clone)]
+pub struct QuorumDriverResponseWithObjects {
```

Comment:
Is there a reason we could not just add objects field to QuorumdriverResponse?

Reply:
this will break client unfortunately

Reply:
How so? Isn't this an internal type used by orchestrator now?

Reply:
you are right. It turns out the old type is not in use after this change. I will merge the two

Reply:
fixed